index.html

```diff
@@ -395,5 +395,40 @@ <h6>Skills</h6>
         </div>
       </section>
     </main>
+    <footer>
+      <div class="container-about-me">
+        <form action="https://formspree.io/f/xoqbwkba" method="post">
+          <ul class="form-list">
+            <li><h2>Contact me</h2></li>
+            <li>
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it
+              </p>
+            </li>
+            <li class="form-list-text">
+              <input type="text" id="username" name="username" placeholder="username" maxlength="30" />
```

Comment:
- [ ] Kindly ensure that all input fields have the `required` attribute to activate the validation on submit so that a user won't submit empty data 💯 .